samples/scala-eventsourced-customer-registry/build.sbt

```diff
@@ -1,6 +1,6 @@
-name := "eventsourced-customer-registry"
+name := "customer-registry"
```

Comment:
why not eventsourced? OK by me though.

Reply:
Alternative would be to put `valueentity` in front of the name in the other project, I'm fine with either.